src/redisraft.c

```diff
@@ -24,28 +25,28 @@ const char *redisraft_loglevels[] = {
     REDISMODULE_LOGLEVEL_DEBUG,
     REDISMODULE_LOGLEVEL_VERBOSE,
     REDISMODULE_LOGLEVEL_NOTICE,
-    REDISMODULE_LOGLEVEL_WARNING
+    REDISMODULE_LOGLEVEL_WARNING,
 };
 
 int redisraft_loglevel_enums[] = {
-        LOG_LEVEL_DEBUG,
-        LOG_LEVEL_VERBOSE,
-        LOG_LEVEL_NOTICE,
-        LOG_LEVEL_WARNING,
-        LOG_LEVEL_COUNT
+    LOG_LEVEL_DEBUG,
+    LOG_LEVEL_VERBOSE,
+    LOG_LEVEL_NOTICE,
+    LOG_LEVEL_WARNING,
+    LOG_LEVEL_COUNT,
```

Comment:
See the trailing comma. Without it, clang-format will not format this way. We need to add it when required.